build.sbt

```diff
@@ -132,7 +132,10 @@ lazy val sconfig = crossProject(JVMPlatform, NativePlatform, JSPlatform)
     Test / envVars ++= Map(
       "testList.0" -> "0",
       "testList.1" -> "1",
-      "testClassesPath" -> (Test / classDirectory).value.getPath
+      "testClassesPath" -> (Test / classDirectory).value.getPath,
+      "SECRET_A" -> "A", // ConfigTest.renderShowEnvVariableValues
+      "SECRET_B" -> "B", // ConfigTest.renderShowEnvVariableValues
+      "SECRET_C" -> "C" // ConfigTest.renderShowEnvVariableValues
```

Comment:
`build.sbt` looks looks heavily changed when compared to lightbend version.

Reply:
Yes, the Lightbend one is Java only. This works for Java, 3 versions of Scala on JVM/JS/Native.

Reply:
My sbt was not reloaded. That's why test was failing XD.
CI green.